src/common/common-types.ts

```diff
@@ -335,6 +335,7 @@ export interface NodeData {
     readonly outputHeight?: OutputHeight;
     readonly nodeWidth?: number;
     readonly isCollapsed?: boolean;
+    readonly nickname?: string;
```

Comment:
Is `nickname` really the right name for this? What if we just call it `name` or `nodeName`?